docs/source/guide/refactor_code/index.rst

```diff
@@ -1,7 +1,156 @@
 Refactoring Code
 ================
 
-.. toctree::
-   :maxdepth: 1
+LineaPy's artifacts store the complete development history in intelligent graph representation,
+and this enables automatic transformation of the development code.
 
-   ../../tutorials/01_refactor_code
+An important application of such transformation is “program slicing” where the development code is
```

Comment:
"Program slicing" is a very technical term in the programming language research literature. In this user documentation, let's stay away from the how and focus on the what, which is code cleanup. "LineaPy's artifacts ... model)." can be removed to help users focus on the problem we solve.